Week1/databases/Homework/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const util = require('util');
+const mysql = require('mysql');
+const {
+  sqlCreateWorld,
+  sqlCreateCountry,
+  sqlCreateCity,
+  sqlWorld,
+  selectQueries,
+  insertCityData,
+  insertCountryData,
+} = require('./query');
+
+const Query = mysql.createConnection('mysql://hyfuser:hyfpassword@192.168.153.132/userdb');
```

Comment:
I guess that's the IP of your own personal computer? Usually what we do is to point to `localhost` or `127.0.0.1` which means it points to the machine where the code is being run.